### ISSUE TITLE
Finetune GPT-2 using LoRA and compare to zero-shot and fine-tuned without LoRA

### ISSUE THREAD

**Issue by varun-suresh** [varun-suresh/ml-practice#4]
Get precision, recall and accuracy binned by review length
